Annotations/Arrow/ArrowDemo/ArrowDemo.sln

```diff
@@ -1,25 +1,25 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.30709.132
+# Visual Studio Version 17
+VisualStudioVersion = 17.10.35122.118

```

Comment:
Change the version as of in our source for .NET8 for all projects

Reply:
Version changing is completed